src/api/function/UrlPurger.cs

```diff
@@ -10,27 +10,27 @@ namespace UrlPurger.function
 
     public class UrlPurger
     {
-        private readonly ILogger _logger;
+        private NLogWrapper logger;
         private readonly AdminApiSettings _adminApiSettings;
 
         public UrlPurger(ILoggerFactory loggerFactory, AdminApiSettings settings)
         {
-            _logger = loggerFactory.CreateLogger<UrlPurger>();
+            logger = new NLogWrapper(LoggerType.UrlPurger, settings);
             _adminApiSettings = settings;
         }
 
         [Function("UrlPurger")]
         public async Task Run([TimerTrigger("0 0 6 * * *", RunOnStartup = true)] MyInfo myTimer)
         {
-            _logger.LogInformation($"C# Timer trigger function executed at: {DateTime.Now}");
-            _logger.LogInformation($"Next timer schedule at: {myTimer.ScheduleStatus.Next}");
+            logger.Log(NLog.LogLevel.Info, "C# Timer trigger function executed at {trigger.current}", DateTime.Now.ToString());
```

Comment:
You could use string interpolation here by prefixing the string with $ (like it was doing previously). Then you wouldn't need the 3rd parameter. Applies to any comment in the PR that uses a variable value in the comment being logged. This will also allow you to remove `.ToString()` from the variable since interpolation will automatically use it without you having to write it.
https://learn.microsoft.com/en-us/dotnet/csharp/language-reference/tokens/interpolated

Reply:
I chose to use the variables to utilize NLog's Structured Logging feature, which I don't believe would work with interpolation. Any variable put in the message string, like `trigger.current` in this case, would be added to the log in that structure. This allowed me to nest related variables like `trigger.current` and `trigger.next`. In the log you'd see the values displayed like this:

```
{
   trigger: {
     current: MM:DD:YYYY 
     next:      MM:DD:YYYY
   }
}
```
With string interpolation I believe the values would just be added to the log message and not to the JSON structure.

Reply:
Ah, that's nice!